test/testground/network/experiments.go

```diff
@@ -0,0 +1,89 @@
+package network
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/celestiaorg/celestia-app/test/util/sdkutil"
+	"github.com/cosmos/cosmos-sdk/codec"
+	coretypes "github.com/tendermint/tendermint/types"
+	"github.com/testground/sdk-go/run"
+	"github.com/testground/sdk-go/runtime"
+)
+
+const (
+	UnboundedBlockSize = "unbounded"
+	ConsistentFill     = "consistent-fill"
+)
+
+
+func fillBlocks(ctx context.Context, runenv *runtime.RunEnv, initCtx *run.InitContext, timeout time.Duration) error {
+	seqs := runenv.IntParam(BlobSequencesParam)
+	size := runenv.IntParam(BlobSizesParam)
+	count := runenv.IntParam(BlobsPerSeqParam)
+
+	cmd := NewRunTxSimCommand("txsim-0", timeout, RunTxSimCommandArgs{
+		BlobSequences: seqs,
+		BlobSize:      size,
+		BlobCount:     count,
+	})
+
+	runenv.RecordMessage("leader: sending txsim command")
+
+	_, err := initCtx.SyncClient.Publish(ctx, CommandTopic, cmd)
+	return err
+}
+
+func (l *Leader) unboundedBlockSize(
+	ctx context.Context,
+	runenv *runtime.RunEnv,
+	initCtx *run.InitContext,
+	cdc codec.Codec,
+	heightStepSize int64,
+) error {
+	err := fillBlocks(ctx, runenv, initCtx, time.Minute*59)
+	if err != nil {
+		return err
+	}
+
+	query := "tm.event = 'NewBlockHeader'"
+	events, err := l.cctx.Client.Subscribe(ctx, "leader", query, 10)
+	if err != nil {
+		return err
+	}
+
+	go func() {
+		blockSize := 2000000
+		blockIncrement := 5000000
```

Comment:
A comment on the size unit (byte or bit) would be nice.

Reply:
good idea, added here 05e325798ef09d1674701e9508ac8398e172d023